lib/tasks/migrations/migrations.rake

```diff
@@ -111,4 +116,8 @@ namespace :migrations do
 
     [start, stop]
   end
+
+  def infer_role_name(name)
```

Comment:
Default role names (user and admin) names on v2 will map their names to v3 configuration,
Other roles will have the same name on v3 but capitalized,